webui/react/src/components/TaskList.tsx

```diff
@@ -68,6 +68,10 @@ const TensorBoardSourceType = {
 
 type TensorBoardSourceType = ValueOf<typeof TensorBoardSourceType>;
 
+interface Props {
+  workspaceId?: number;
```

Comment:
pass `workspaceId` for useSettings and APIs of `getCommands`, `getJupyterLabs`, `getShells`, `getTensorBoards`
when its `undefined`, that means its global (fetch all)